client/my-sites/plugins/index.web.js

```diff
@@ -93,7 +93,7 @@ export default function ( router ) {
 	router( `/${ langParam }/plugins/plans`, notFound, makeLayout );
 
 	router(
-		`/${ langParam }/plugins/plans/:site`,
+		`/${ langParam }/plugins/plans/:intervalType(yearly|monthly)/:site`,
```

Comment:
```suggestion
		`/${ langParam }/plugins/plans/:plugin/:intervalType(yearly|monthly)/:site`,
```

To add plugin we'll need to use this path so we can set a corresponding base path for the toggle links to work.